packages/back-end/src/services/orb.ts

```diff
@@ -0,0 +1,56 @@
+import Orb from "orb-billing";
+import { addUsageAlert } from "back-end/src/models/UsageAlertsModel";
+import { ORB_API_KEY } from "back-end/src/util/secrets";
+import { SubscriptionUsageExceededEvent } from "back-end/src/types/Orb";
+import { getOrganizationById } from "./organizations";
+
+export const orb = new Orb({
+  apiKey: ORB_API_KEY,
+});
+
+function findMaxFreeThreshold(priceData: Orb.Prices.Price.TieredPrice) {
+  return priceData.tiered_config.tiers.find(
+    (tier) => tier.first_unit === 0.0 && tier.unit_amount === "0.00"
+  )?.last_unit;
```

Comment:
This logic will have to change for free plans since they won't use tiered pricing